README.md

```diff
@@ -496,14 +505,16 @@ We have a small collection of algorithms, one for every day of the month. Scroll
     8                14
     ```
   - **Resources**  
-    [Dijkstra's Algorithm](https://en.wikipedia.org/wiki/Dijkstra%27s_algorithm)
+    - [What is Dijkstra's Algorithm?](https://en.wikipedia.org/wiki/Dijkstra%27s_algorithm)
+    - [Visualisation of Dijkstra's Algorithm #1](https://visualgo.net/en/sssp)
+    - [Visualisation of Dijkstra's Algorithm #2](https://www.cs.usfca.edu/~galles/visualization/Dijkstra.html)
 
 
 ## **December 21 - Currency Convertor**
   - **Problem**
       -  Calculate the currency exchange rates between two user input countries using the attached **.csv** file in the reference.
-      - ``Note:`` All the given ``VALUES`` in the csv file are a countries equivalent value in USD.
-      - ``Optional:`` Generate a complete list of currency units for conversion
+      - ``Note:`` All the given ``VALUES`` in the csv file are a country's equivalent value in USD.
+      - ``Optional:`` Also, generate a complete list of currency exchange rates between the input currency and the world currencies. 
```

Comment:
So for instance if I give the input as India, the program should give a list of exchange rates for India? Like USA = 70, Malaysia = 17 and so on?

Reply:
Yess!

Reply:
Alright, just put that in as an explanation for it. It's a bit vague as it is now.